polaris-core/src/main/java/org/apache/polaris/core/storage/cache/StorageCredentialCacheKey.java

```diff
@@ -44,18 +42,13 @@ public class StorageCredentialCacheKey {
 
   private final Set<String> allowedWriteLocations;
 
-  /**
-   * The callContext is passed to be used to fetch subscoped creds, but is not used to hash/equals
-   * as part of the cache key.
-   */
-  private @Nullable PolarisCallContext callContext;
```

Comment:
I suspect the removal of this field is the key of this PR?

It totally makes sense to not keep per-request state around.

Reply:
yes but also removal of the `realmContext` field is important as well, as it is request scope but was being used inside the expiry logic of the cache.